packages/e2e/docker-compose-dev.yml

```diff
@@ -1,9 +1,7 @@
-version: "3.9"
-
 services:
   http-server:
     volumes:
-      - ..:/app/packages
+      - ../..:/app
```

Comment:
These changes are in the wrong commit

Reply:
18b0bc171 plus 8643ce86a should solve this